source/ScreensaverHero.hx

```diff
@@ -0,0 +1,92 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxRandom;
+import flixel.util.FlxSpriteUtil;
+
+class ScreensaverHero extends FlxSprite
+{
+	public var direction:Int;
+
+	public function new(x:Int, y:Int)
+	{
+		super(x, y);
+		var randomDirection:FlxRandom = new FlxRandom();
+		this.direction = randomDirection.int(0, 7);
+	}
```

Comment:
This is missing the call to `makeGraphic()` mentioned in the assignment specifications:
> 3. Inside the ScreensaverHero's constructor, use makeGraphic() to give it a simple rectangle graphic.